chapters/location_component_interface.adoc

```diff
@@ -146,14 +144,21 @@ From a `Location`/`Component` point-of-view looks like
 layout(location = 0) in vec2 a[3];
 ----
 
-As stated above, arrays consume the whole `Location` so the following is **not** allowed.
+Just like vector, depending on the size, it may not consume the whole `Location`. This means the following is allowed.
```

Comment:
```suggestion
which in turn is stored identically to

[source,glsl]
----
// Each vector consumes only the first 2 components of each location
layout(location = 0) vec2 a0;
layout(location = 1) vec2 a1;
layout(location = 2) vec2 a2;
----

This means the following is allowed:
```

Also the "Arrays consume whole location" comment above is wrong - where did that come from?